spec/requests/api/v1/posters_request_spec.rb

```diff
@@ -132,11 +132,12 @@
     original_name = poster.name
     new_name = test_params[:name]
 
-    patch "/api/vi/posters/#{poster.id}", params: test_params.to_json, headers: { "Content-Type" => "application/json" }
-
+    patch "/api/v1/posters/#{poster.id}", params: test_params.to_json, headers: { "Content-Type" => "application/json" }
+    new_poster = JSON.parse(response.body, symbolize_names: true)
```

Comment:
adding this `new_poster` variable will allow us to compare the change before and after the `patch` occurs in our test